pkg/virtctl/expose/expose.go

```diff
@@ -291,9 +291,12 @@ func convertIPFamily(strIPFamily string) ([]v1.IPFamily, error) {
 	}
 }
 
-func convertIPFamilyPolicy(strIPFamilyPolicy string) (v1.IPFamilyPolicyType, error) {
+func convertIPFamilyPolicy(strIPFamilyPolicy string, ipFamilies []v1.IPFamily) (v1.IPFamilyPolicyType, error) {
 	switch strings.ToLower(strIPFamilyPolicy) {
 	case "":
+		if len(ipFamilies) > 1 {
+			return v1.IPFamilyPolicyPreferDualStack, nil
```

Comment:
Probably `IPFamilyPolicyRequireDualStack` is more like it ...

Reply:
Can we test this at the k8s-1.23 provider ?

Reply:
[PR](https://github.com/kubevirt/kubevirt/pull/6947) was merged, so I guess so. @dhiller how can we test drive this ?